_posts/2023-05-19-story.md

```diff
@@ -0,0 +1,608 @@
+
+Talia squinted at the shaky video feed. "Eva, CAPCOM. Can you describe what you see?"
+
+"The outside has these metallic looking hexagonal fish scales with dark little creases between them. The whole surface seems uniform. I don't see any thrusters or antennas. It doesn't make sense. Talia, you getting anything on the sensors?"
+
+"Still nothing. It's quiet, as it promised."
+
+"I'm going to maneuver to look in the inside."
+
+She floated over to the inside of the tube. It looked pitch black against the harsh shadow of the outer-space sun, with no atmosphere to provide indirect lighting. She flipped on her suit lights. 
+
+"The inside looks similar, that metallic honeycomb fish-scale surface, whatever you want to call it. Wait, some of the scales have different colors. There's bulging in spots, like blisters. Lines streaking across. It almost looks...injured. Maybe it took a beating on the way over here." 
+
+She panned around the inside with her suit light. "Talia, any signals yet?"
+
+"Still nothing. If I get something, you'll be the first to know."
+
+Talia muted herself to get an update from the Commander. Talia unmuted and relayed it over the radio, "Eva, we're seeing lots of hostile activity from the EAC. Get ready to evac if they try something."
+
+"Received. Here goes nothing."
+
+She slowly reached out her sausage-like gloved finger, the pressurized suit keeping her fingers inflated. She touched the surface then withdrew her hand immediately, waiting for a reaction. Nothing. She ran her fingers along the surface. 
+
+"It feels smooth with some undulating cracks. I'm not getting any sign of life." 
+
+Talia replied. "It said 'physical contact'. Maybe that's sound waves. Can you listen to it?"
+
+"I'm going to put my helmet up to it."
+
+She held on to the curved lip at the edge of the hole, one hand on the dark inside, the other on the sunny outside. She rotated her whole body forward until her faceplate made a _thud_ against the object's surface. 
+
+"Hello. This is Eva. Remember? We communicated a while ago. I'm here."
+
+She waited. 
+
+"I'm not getting anything," she said over the radio. 
+
+Suddenly, she heard high pitched noises, almost like scratching. It sounded like a dentist's cleaning instrument moving around inside her helmet.
+
+"You hearing that?" Eva said. 
+
+<!-- Forager -->
+
+"Yes, it sound scratchy." 
+
+She heard a voice. A faint voice. It sounded like a girl's voice. 
+
+"way afar afar afar hello hello afar afar hello hello"
+
+It increased in volume. 
+
+"Yes? I hear you."
+
+"Eva makes..." it said. She couldn't hear the rest.
+
+"I can't hear. It's too quiet." 
+
+Louder this time. "Eva makes physical contact," it said.
+
+Eva recoiled, pulling her helmet away so it wouldn't hear.
+
+"Ok, I think it's talking. Also it sounds like...me. When I was a kid."
+
+She made contact with her helmet again. "What's your name?"
+
+"The name's Bond, James Bond."
```

Comment:
Aliens communicating in pop culture references is also a cliche. The voice sounds like Eva as a little girl -- does the object have access to Eva's thoughts and memories? If so, what if it replied with the name of a childhood imaginary friend or toy, or an old nickname? Or an in-joke that no one else would know? In any case, what is the object trying to do by replying with the James Bond line? Confuse her, amuse her, demonstrate awareness of her culture, something else?